### ISSUE TITLE
Feature: more languages

### ISSUE THREAD

**Issue by timethy96** [TheMoMStudio/swzpln.de#5]
Language chooser would be needed.
Which back-end for multiple languages?

**Comment by Vestibule** [comment]
A backend is not necessarily needed to support multiple languages.
A simple VueJS application with i18n could do the trick. The implementation would need a bit of rework of the codebase but it could be worth it.
I have experience in building VueJS application, I could take care of it.

**Comment by timethy96** [comment]
I don't think that the translation of these 2 or 3 sentences justifies this rather big change in the codebase. (It could even easily be achieved by setting a cookie and serving the text via php)
The project aims to be minimalistic and easy to understand (which is admittedly rather not the case at the moment).
If we want to rewrite the codebase with VueJS or any other library, the question we have to ask is, if it would contribute to said goals.

**Comment by Vestibule** [comment]
I agree implementing VueJS for just a few sentences is a bit overkill. I still think it could be done in the frontend, without the overhead of implementing localisation in PHP.

**Comment by timethy96** [comment]
closed with release v1.00-RC1
added english language
